pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ReaderBuilder.java

```diff
@@ -367,4 +367,15 @@
      */
     ReaderBuilder<T> expireTimeOfIncompleteChunkedMessage(long duration, TimeUnit unit);
 
+    /**
+     * Start the reader in a paused state. When enabled, the reader does not immediately fetch messages when
+     * {@link #create()} is called. Instead, the reader waits to fetch messages until {@link Reader#resume()} is
+     * called.
+     * <p/>
+     * See also {@link Reader#pause()}.
+     *
+     * @default false
+     */
+    ReaderBuilder<T> startPaused(boolean paused);
```

Comment:
startPaused -> paused  is better ?  I'm not sure. 
@codelipenghui  What about you?

Reply:
There is already a setting `startPaused` in the consumer. https://github.com/apache/pulsar/blob/bfa9e7872d0f693dfdbe38df2ae27142ae457894/pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java#L800 
 I think we need to keep it the same.

Reply:
Yes, keep it the same